storage/Devices/Bcache.cc

```diff
@@ -124,6 +166,13 @@ namespace storage
     }
 
 
+    void
+    Bcache::reassign_numbers(Devicegraph* devicegraph)
+    {
```

Comment:
It can be only internal. I think it can be useful from outside when needed to fix numbers even when no device removal is done. E.g. when construction names from autoyast profile and if there will be some holes, then this method will fix it.

Reply:
OK, that is a good reason.